site/update-image.md

```diff
@@ -72,11 +72,28 @@ increase the size of the image significantly, and is not recommended.
 ```dockerfile
 [after-fmw-install]
 RUN rm /some/dir/unnecessary-file
+COPY --chown=oracle:oracle files/my_additional_file.txt /u01
 
 [final-build-commands]
 LABEL owner="middleware team"
 ```
 
+#### `--additionalBuildFiles`
+
+This option provides a way to supply additional files to the image build command.
+All provided files are copied directly under the `files` subfolder of the build context.  
+To get those files into the image, additional build commands must be provided using the `additionalBuildCommands` options.
+Access to these files using a build command such as COPY or ADD should use the original filename 
+with the folder prefix, `files/`.  For example, if the 
+original file was provided as `--additionalBuildFiles /scratch/test1/convenience.sh`, the Docker build command `COPY`
+provided in `--additionalBuildCommands` should look like 
+`COPY --chown=oracle:oracle files/convenience.sh /my/internal/image/location`.  Since Image Tool uses multi-stage 
```

Comment:
Since Image Tool uses -> Because the Image Tool uses (Also, I would suggest that you start a new paragraph with this sentence.)